NineChronicles.Headless.Tests/NineChronicles.Headless.Tests.csproj

```diff
@@ -6,6 +6,7 @@
     <IsPackable>false</IsPackable>
     <IsTestProject>true</IsTestProject>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <NoWarn>$(NoWarn);CS8602;CS8604</NoWarn>
```

Comment:
I'm not sure about just muting the warnings :thinking:

Reply:
@U-lis Yeah, Me too.😅 At least for now, I'm trying to passing CI because it is a test project.
After then remove these no-warn settings and fix test codes separately.